job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -54,6 +54,16 @@ parameters:
     pattern: '^[^.,:;*?!|()]{1,48}$'
 
 paths:
+  /ping:
+    get:
+      tags:
+        - Jobs
+      summary: Ping Service
+      description: Ping Service
+      operationId: ping
+      responses:
+        '200':
+          description: OK
```

Comment:
Whats the expected response code here if the ping fails

Reply:
Added 500 error code